borg/helpers.py

```diff
@@ -212,7 +212,11 @@ def show_progress(self, item=None, final=False, stream=None, dt=None):
 
 def get_keys_dir():
     """Determine where to repository keys and cache"""
-    xdg_config = os.environ.get('XDG_CONFIG_HOME', os.path.join(os.path.expanduser('~'), '.config'))
+    if os.environ.get('HOME'):
+        home_dir = os.environ.get('HOME')
+    else:
+        home_dir = os.path.expanduser('~%s' % os.environ.get('USER'))
```

Comment:
btw, we have other code that uses the home dir (e.g. cache location), so does that need the same change also?

I am not sure why this needed fixing here, can you explain?